redisson/src/main/java/org/redisson/RedissonLock.java

```diff
@@ -249,16 +249,8 @@ void cancelExpirationRenewal() {
         internalLockLeaseTime = unit.toMillis(leaseTime);
 
         return commandExecutor.evalWriteAsync(getName(), LongCodec.INSTANCE, command,
-                  "if (redis.call('exists', KEYS[1]) == 0) then " +
-                      "redis.call('hset', KEYS[1], ARGV[2], 1); " +
-                      "redis.call('pexpire', KEYS[1], ARGV[1]); " +
-                      "return nil; " +
```

Comment:
@Aslan11 

return statements are missed

Reply:
I'm not a java expert but you see the optimization I'm making here correct? you do not need to initialize the hash key with hset. you can do so with hincrby. the if statement here is unnecessary

Reply:
@Aslan11 the logic is broken, you cannot just return nil, `return redis.call` will never be invoked.